foundation/foundation-mda/src/main/resources/templates/cucumber.spark.test.base.harness.java.vm

```diff
@@ -92,7 +92,11 @@ public abstract class SparkTestBaseHarness implements EventListener {
 
         for (Map.Entry<String, String> entry : sparkApplicationConfig.spec().sparkConf().entrySet()) {
             logger.warn(entry.getKey() + " " + entry.getValue());
-            builder = builder.config(entry.getKey(), entry.getValue());
+            if (!entry.getKey().equalsIgnoreCase("spark.jars.ivy")) {
+                builder = builder.config(entry.getKey(), entry.getValue());
+            } else {
+                logger.warn(String.format("Skip spark config: %s - %s", entry.getKey(), entry.getValue()));
```

Comment:
A: Recommend info here since we expect this value to always exist and be skipped